components/style/themes/default.less

```diff
@@ -391,7 +394,9 @@
 @menu-dark-submenu-bg: #000c17;
 @menu-dark-highlight-color: #fff;
 @menu-dark-item-active-bg: @primary-color;
-
+@menu-dark-selected-item-icon-color: @white;
+@menu-dark-selected-item-text-color: @white;
```

Comment:
Can we merge `menu-dark-selected-item-icon-color` & `menu-dark-selected-item-text-color`?

Reply:
why not?